src/LayerEditor/ui/panels/regions_panel.py

```diff
@@ -449,7 +432,7 @@ def _update_region_content(self):
         self.wg_dims.setText(AddRegionDlg.REGION_DESCRIPTION_DIM[region.dim])
         self.wg_boundary.setChecked(region.boundary)
         self.wg_notused.setChecked(region.not_used)
-        self.wg_mesh_step_edit.setText("{:8.4g}".format(region.mesh_step))
+        self.wg_mesh_step_edit.setText("{:8.4g}".format(region.mesh_step).replace(" ", ""))
```

Comment:
??

Reply:
I misunderstood the syntax, and fixed it. Done